lib/functoria_misc.ml

```diff
@@ -293,7 +83,7 @@ module Codegen = struct
         days.(time.Unix.tm_wday) time.Unix.tm_mday
         months.(time.Unix.tm_mon) (time.Unix.tm_year+1900)
         time.Unix.tm_hour time.Unix.tm_min time.Unix.tm_sec in
-    Format.sprintf "Generated by %s (%s)." name date
+    Format.sprintf "Generated by functoria (%s)." date
```

Comment:
I was asked specifically not to put functoria here. The actual application (aka mirage, or the unikernel) should put its name here. Not sure how @samoht 's position has evolved on the matter. :)